spec/system/purchase_system_spec.rb

```diff
@@ -47,6 +47,15 @@
           expect(page).to have_text(dollar_value(purchases.sum(&:amount_spent_in_cents)))
           expect(page).to have_text(dollar_value(3579))
         end
+
+        it "user sees FMV column" do
```

Comment:
Can this be a request spec instead of a system spec? It's a lighter load.

Reply:
Sure thing. I was following the pattern of the test above. Do you want me to move that one too? Or maybe just add a to-do comment to keep this PR scoped to this change?